src/screens/SecretIconsScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../components';
+import { colors, Icon, normalize } from '../config';
+
+type TIconItem = {
+  iconName: string;
+};
+
+const iconNames = Object.keys(Icon);
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
```

Comment:
TypeScript complains here:

![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/213f65aa-bbc9-4e66-a733-0114fba71e64)

Please take some time to describe the situation for yourself and try to find the solution.

Reply:
should be solved now in the new commit [29186de](https://github.com/smart-village-solutions/smart-village-app-app/pull/914/commits/29186de714d420a15bf9284a4e17cbc553609203)

Reply:
well done, how did you find out about the solution?